src/bosh-softlayer-cpi/action/create_vm_test.go

```diff
@@ -78,6 +79,8 @@ var _ = Describe("CreateVM", func() {
 				fakeStemcellFinder.FindByIdReturns(fakeStemcell, nil)
 				fakeCreatorProvider.GetReturns(fakeVmCreator)
 				fakeVmCreator.CreateReturns(fakeVm, nil)
+				Expect(helper.LengthOfHostName).ToNot(Equal(64))
```

Comment:
I would like to see a test that shows the length of VMs to be less than 64 since the point is not to be longer.